src/main/webapp/confirmation.js

```diff
@@ -12,86 +12,95 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** Gets a list of the user's scheduled sessions from the server and displays them on the page. */
-function getScheduledSessions() {
-    return getScheduledSessionsHelper(window);
-}
-
-//Helper function for getScheduledSessions, used for testing
-async function getScheduledSessionsHelper(window) {
-    await fetch('/confirmation', {method: 'GET'}).then((response) => {
+/** Creates a calendar with the Charts API and renders it on the page  */
+function createCalendar() {
+    fetch('/confirmation', {method: 'GET'}).then((response) => {
         //if the student is not the current user or not signed in
         if(response.redirected) {
             window.location.href = response.url;
-            alert("You must be signed in to view upcoming session.");
+            alert("You must be signed in to manage sessions.");
             return [];
         }
         return response.json();
+    }).then(async (scheduledSessions) => {
+        const container = document.getElementById('calendar');
         
-    }).then((scheduledSessions) => {
         if(scheduledSessions.error) {
             var message = document.createElement("p");
             p.innerText = scheduledSessions.error;
-            document.getElementById('timeslots').appendChild(message);
+            document.getElementById('calendar').appendChild(message);
             return;
         }
 
-        if (Object.keys(scheduledSessions).length != 0) {
-            scheduledSessions.forEach((scheduledSession) => {
-                document.getElementById('scheduledSessions').appendChild(createScheduledSessionBox(scheduledSession));
-            });
-        } else {
-            var sessionsContainer = document.getElementById('scheduledSessions');
+        // Don't create a calendar if there are no scheduled sessions
+        if (scheduledSessions === undefined || scheduledSessions.length == 0) {
             var errorMessage = document.createElement("p");
             errorMessage.innerText = "This user does not have any scheduled tutoring sessions.";
-            sessionsContainer.appendChild(errorMessage);
+            container.appendChild(errorMessage);
             return;
         }
-    });
-}
-
-
-function createScheduledSessionBox(scheduledSession) {
-
-    var months = [ "January", "February", "March", "April", "May", "June", 
-           "July", "August", "September", "October", "November", "December" ];
-
-    const scheduledSessionElement = document.createElement('li');
-    scheduledSessionElement.className = 'list-group-item';
+        
+        const chart = new google.visualization.Timeline(container);
```

Comment:
Maybe add the part that constructs the calendar as a separate function from the part that fetches confirmation